src/frontend/packages/lib_components/src/hooks/stores/useJwt/index.ts

```diff
@@ -40,38 +41,38 @@ const localStore = create<JwtStoreInterface>((set, get) => ({
   },
 }));
 
-const persistentStore = create<JwtStoreInterface>()(
-  persist(
-    (set, get) => ({
-      jwt: undefined,
-      refreshJwt: undefined,
-      internalDecodedJwt: undefined,
-      jwtCreateTimestamp: undefined,
-      setJwt: (jwt) => set((state) => ({ ...state, jwt })),
-      setRefreshJwt: (refreshJwt) => set((state) => ({ ...state, refreshJwt })),
-      getDecodedJwt: () => {
-        const currentValue = get().internalDecodedJwt;
-        if (currentValue) {
-          return currentValue;
-        }
+const persistentStore = create<JwtStoreInterface>((set, get) => ({
+  get jwt() {
+    return localStorage.getItem(JWT_KEY) || undefined;
+  },
+  get refreshJwt() {
+    return localStorage.getItem(REFRESH_JWT_KEY) || undefined;
+  },
+  internalDecodedJwt: undefined,
+  jwtCreateTimestamp: undefined,
```

Comment:
```suggestion
```

This property is not in the Interface anymore. You can also remove it in the non persistent store.